business_requirement_deliverable/models/business.py

```diff
@@ -0,0 +1,279 @@
+# © 2016-2019 Elico Corp (https://www.elico-corp.com).
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from odoo import api, fields, models, _
+from odoo.exceptions import UserError
+from odoo.addons import decimal_precision as dp
+
+
+class BusinessRequirementDeliverable(models.Model):
+    _name = "business.requirement.deliverable"
+    _inherit = ["mail.thread", "mail.activity.mixin"]
+    _description = "Business Requirement Deliverable"
+
+    sequence = fields.Integer('Sequence')
+    state = fields.Selection(
+        related='business_requirement_id.state',
+        selection=[('draft', 'Draft'),
+                   ('confirmed', 'Confirmed'),
+                   ('approved', 'Approved'),
+                   ('stakeholder_approval', 'Stakeholder Approval'),
+                   ('in_progress', 'In progress'),
+                   ('done', 'Done'),
+                   ('cancel', 'Cancel'),
+                   ('drop', 'Drop'),
+                   ],
+        store=True,
```

Comment:
Why is this needed? It requires `readonly = True`

Reply:
Is to control others fields on views to be editable but you are right need to be readonly true